go.mod

```
@@ -0,0 +1,3 @@
+module github.com/sudhirj/strobe
```

```suggestion
module github.com/sudhirj/strobe/v1
```

The module path can be extended with v1 or some other string to not break existing code that depends on the latest library without vendoring.